site/composer.lock

```diff
@@ -63,30 +63,34 @@
                 "markdown",
```

Comment:
not sure why you're updating this file in this PR if you're not also updating the composer.json file. Should revert this change unless there is a necessary change to composer.json.